### PR TITLE
Awesome books: a complete website with navigation

### PR DESCRIPTION
**Pull request by ad9311**
### Greetings 👋🏽 

In this PR we present the following changes:

- Three new pages in a single page: book list, add a new book and contact info.
- Full date using luxon.
- Overall improvment to visuals.

**Note:** Please keep in mind that luxon only works if you are launching the application in the live server.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,41 +1,76 @@
...
+  <div class="container-md d-flex flex-row-reverse py-1">
+    <p id="date"></p>
+  </div>
+  <main>
+    <section id="section-list" class="py-4">
+      <h1 class="text-center mb-5">All Awesome Books</h1>
+      <div class="book-list">
+        <ul id="books" class="p-0">
+        </ul>
+      </div>
```
Comment (rloterh) [inline index.html:32]: - [ ] Your booklist container is overlapping the the footer. We actually want you design to as much as possible mimic the provided [wireframe template](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m4_full_website.md#project-requirements) design. As best practice it is a  good idea to leave the booklist container height as not set, and therefore it's height will grow **automatically** as each book is added. Also, the footer should be pushed downwards **automatically** as the height of the booklist container grows. Kindly ensure to improve upon the above aspects.

**Your design:**
image

File: index.html [hunk 2]
```diff
@@ -1,41 +1,76 @@
...
+</head>
+
+<body>
+  <nav class=" nav d-flex justify-content-between container-md border border-dark mt-3 py-2 align-items-center">
+    <p class="px-3 m-0">Awesome Books</p>
+    <ul class="nav-list d-flex justify-content-between m-0">
+      <li><a id="list" class="text-decoration-none px-2">List </a> |</li>
+      <li class="bg-transparent"><a id="new" class="text-decoration-none px-2">Add new </a> |</li>
+      <li><a class="text-decoration-none px-2" id="contact">Contact </a></li>
+    </ul>
```
Comment (rloterh) [inline index.html:21]: - [ ] To enhance the user experience of this design kindly ensure that an active menu link is highlighted( should have color change or change in opacity  of the active menu item). In other words, the current active page should have the corresponding menu item highlighted.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Angel & Chibuike, 

You did extremely great on this project!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚆  

## Good Points 👍  
- No linter errors


## Required Changes ♻️
- [x] _Kindly check the comments under the review._


Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ad9311** [comment]
We have made the following changes to comply with the previous comments:

- The footer does not longer overlaps the book list.
- The book list does not longer have a set height. It grows as new books are added.
- The footer gets pushed down as the book list grows.
- When the navigation links are clicked they change color and get bolder. This indicates the users in which section they are.

**Review by Lameck1 — APPROVED** [review]
Hi, @ad9311 & @Chike1990,

I confirm that the concerns of the previous review have been resolve. Your project is now complete!  It's time to merge it :shipit:

Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**